src/Toolbox.php

```diff
@@ -2618,21 +2618,21 @@ public static function convertTagToImage($content_text, CommonDBTM $item, $doc_d
                         $document->getFromDB($id)
                         && strpos($document->fields['mime'], 'image/') !== false
                     ) {
-                        // append itil object reference in image link
-                        $itil_object = null;
-                        if ($item instanceof CommonITILObject) {
-                            $itil_object = $item;
-                        } else if (
-                            isset($item->input['_job'])
+                        // append object reference in image link
+                        $linked_object = null;
+                        if (!($item instanceof CommonITILObject)
+                              && isset($item->input['_job'])
                               && $item->input['_job'] instanceof CommonITILObject
                         ) {
```

Comment:
From my understanding, this case matches ITILFollowup and ITILSolution

I think this is a inclear and risky expression. ITILFollowup and ITILSolution both inherit from CommonDBChild. I think we could make this better if we introduce a new (abstract) class as a parent of those itemtypes. As they show in a timeline, maybe a CommonITILTimeline class could be helpful (the classes have some identical code to deduplicate).